gum/guminterceptor.c

```diff
@@ -1939,9 +2054,16 @@ interceptor_thread_context_new (void)
   context->stack = g_array_sized_new (FALSE, TRUE,
       sizeof (GumInvocationStackEntry), GUM_MAX_CALL_DEPTH);
 
+
```

Comment:
Style nitpick: For consistency, two or more consecutive blank lines shouldn't be used.